src/libraries/Common/tests/TestUtilities/System/Buffers/BoundedMemory.Windows.cs

```diff
@@ -305,7 +305,9 @@ private static class UnsafeNativeMethods
 
             // https://msdn.microsoft.com/en-us/library/windows/desktop/aa366892(v=vs.85).aspx
             [DllImport(KERNEL32_LIB, CallingConvention = CallingConvention.Winapi, SetLastError = true)]
+#if FEATURE_CER
             [ReliabilityContract(Consistency.WillNotCorruptState, Cer.Success)]
+#endif
```

Comment:
Why do we need this attribute at all?

Reply:
I wasn't positive whether or not there was a build that would utilize it; but I'll remove it and then I can remove the constant from the csproj too.